### DISCUSSION TITLE
SEB text displaying issue

### DISCUSSION THREAD

**Discussion by 4DTrouble** [SafeExamBrowser/seb-win-refactoring#390] [Q&A]
Hi all

We've been using SEB for french Exams since 2020. This year we've had a few users where some text wouldn't show proprely (see Screenshot below -> Top Right).
![B35DAE4F-F97B-4995-A205-806FC55BEBEC](https://user-images.githubusercontent.com/103124327/162219213-eccf2289-6521-4c0f-98ca-0ecc996aad7b.jpeg)

Has anybody experienced something like this before?

I can't provide any logs at the moment. I'm still trying to replicate it. However I can give the following infos:

- SEB 3.3.2.413
- We are running virtual Machines (VMware)
- The SEB Konfig allows VMs
- The exam is in french is being hosted by a french company
- once another user logs into the same VM the error does not occur
- if the affected user changes the VM the error does not appear on the new VM

Thanks and regards

**Comment by dbuechel** [comment]
Looking from afar, this rather looks like an encoding issue within the web application itself. To nonetheless be able to determine whether it is a SEB issue or not, please try to also replicate the issue in Google Chrome / Chromium and the CEF sample application (https://cef-builds.spotifycdn.com/index.html#windows64). Only if it does not replicate in both, then it might indeed be a SEB issue.

**Comment by 4DTrouble** [comment]
Unfortunately the issue never arose again and we weren't able to replicate it after testing with hundreds of accounts and different workstations. We will go through the process, should the issue appear again.
